stanford-earth/matson
# EARTH-1070: Fix Byline floating problem

# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/EARTH-1070_
Fixes the floating byline or author field.

It also adds some formatting clean up I was doing on a separate file. 

# Needed By (Date)
- Soon

# Urgency
- Fixed on prod, need to put into code.


# Steps to Test

- Checkout this branch
- Navigate to http://se3blt.lndo.site/news/when-killer-climate-catastrophe-struck-worlds-oceans
- Inspect the author (byline) filed
- Verify you see
```
section-news .field-s-news-author {
+  display: block; 
}
```

# Affected Projects or Products
- Earth (https://earth.stanford.edu)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/EARTH-1070

## Related PRs

## More Information
I also alphabetized the file

## Folks to notify

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
